sklearn/ensemble/tests/test_iforest.py

```diff
@@ -143,9 +143,9 @@ def test_max_samples_attribute():
     assert clf.max_samples_ == 0.4 * X.shape[0]
 
 
-def test_iforest_parallel_regression():
+def test_iforest_parallel_regression(global_random_seed):
```

Comment:
In this test, IsolationForest is still instanciated with random_state=0. I think it would make sense to set `random_state=global_random_seed`